README.md

```diff
@@ -54,7 +56,8 @@ Using Ruby...
 Write your code here...
 
 ```ruby
-# code here
+belle = residents.slice[1]
+guests << belle
```

Comment:
`slice` won't update the original array so want to call `town[:residents].slice!(1)`

also, `slice` is a little weird (in order to provide functionality that looks like array access with brackets), we can use the bracket notation `town[:residence][1]` which is sugar for calling the method with the desired index as an argument which will use parens `town[:residence].slice(1)`. In practice we always use the former